connect-widget/app/src/shared/api.ts

```diff
@@ -25,24 +26,39 @@ export const getApi = () => {
 export function getApiToken(searchParams: URLSearchParams): string {
   const apiToken = searchParams.get(Constants.TOKEN_PARAM);
   if (!apiToken) {
-    throw new Error(
-      `Missing query param ${Constants.TOKEN_PARAM}! To learn more go to the Connect Widget overview in our documentation.`
+    throw new NoTokenError(
+      `Missing '${Constants.TOKEN_PARAM}' query parameter! To learn more, go to the Connect Widget overview in our documentation.`
```

Comment:
What do you think leave all these messages on the *Error classes? They already have the title, might as well just move all messaging there.

Could pass `TOKEN_PARAM` on the constructor in this case.

Reply:
You mean delete all the custom error classes?

Reply:
No, I mean the custom error classes already have the title there, so they could also provide the default message, so we don't need to set those when we throw an error.

Reply:
Ohhh I see what you mean! Change on the way! 🚄